.travis/linux.win.download.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -e
+
+CACHE_DIR=$TRAVIS_BUILD_DIR/apt_mingw_cache/$1
+mkdir -p $CACHE_DIR
+
+pushd $CACHE_DIR
```

Comment:
Ah! I pop back to the previous directory so I don't break scripts running this file.